enterprise/cmd/frontend/internal/compute/streaming/stream.go

```diff
@@ -109,15 +109,30 @@ LOOP:
 
 	matchesFlush()
 
-	if err = getErr(); err != nil {
+	alert, err := getResults()
+	if err != nil {
 		_ = eventWriter.Event("error", streamhttp.EventError{Message: err.Error()})
 		return
 	}
 
 	if err := ctx.Err(); errors.Is(err, context.DeadlineExceeded) {
 		_ = eventWriter.Event("alert", streamhttp.EventAlert{
-			Title:       "Heads up",
-			Description: "This data is incomplete! We ran this query for 1 minute and we'll need more time to compute all the results. Ask in #compute for more Compute Credits™",
+			Title:       "Incomplete data",
+			Description: "This data is incomplete! We ran this query for 1 minute and we'll need more time to compute all the results. Please reach out to support@sourcegraph.com if it persists",
```

Comment:
```suggestion
			Description: "This data is incomplete! We ran this query for 1 minute and we'd need more time to compute all the results. This isn't supported yet, so please reach out to support@sourcegraph.com if you're interested in running longer queries.",
```

Slight rephrase to make it explicit this isn't supported (rather than something that someone may think it's transient--it's always going to persist for the same long running queries :-))